coffee-chats/src/main/java/com/google/step/coffee/entity/ChatRequestBuilder.java

```diff
@@ -0,0 +1,67 @@
+package com.google.step.coffee.entity;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class ChatRequestBuilder {
+  private static final int MAX_PARTICIPANTS = 5;
```

Comment:
I thought it's 4

Reply:
Similar to the comment above, I believe I need to write some clarifying comments. The max people in a single scheduled call is 5, thus the maximum other participants that a request can ask for is 4. I will attempt to document and clarify this better.